tools/extension_point_identifiers_parser/extension_point_identifiers_parser.py

```diff
@@ -20,14 +20,24 @@ def _main(
         )
         sys.exit(1)
 
+    extensionPointIdentifierKeys = [
+        ("NSExtension", "NSExtensionPointIdentifier"),
+        ("EXAppExtensionAttributes", "EXExtensionPointIdentifier")
+    ]
```

Comment:
Let's pull this out to a `_EXTENSION_POINT_IDENTIFIER_KEYS` constant.

Reply:
Done